EXPhotoViewer/EXPhotoViewer.m

```diff
@@ -98,6 +98,7 @@ - (void)show {
     self.originalImageRect = [self.originalImageView convertRect:self.originalImageView.bounds toView:self.view];
 
     self.theImageView.frame = self.originalImageRect;
+    self.theImageView.contentMode = self.originalImageView.contentMode;
```

Comment:
I like the idea behind this, but I think it would be better suited to go in the `-loadView` method replacing line 59:
```
imageView.contentMode = UIViewContentModeScaleAspectFill;
```